tutorials/tts/FastPitch_Adapter_Finetuning.ipynb

```diff
@@ -0,0 +1,877 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "df77c416",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!cd {codedir} && python scripts/dataset_processing/tts/extract_sup_data.py \\\n",
+    "    manifest_filepath={train_manifest} \\\n",
+    "    sup_data_path={suppdir} \\\n",
+    "    dataset.sample_rate={sample_rate} \\\n",
+    "    dataset.n_fft=2048 \\\n",
+    "    dataset.win_length=2048 \\\n",
+    "    dataset.hop_length=512"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bafbf280",
+   "metadata": {},
+   "source": [
+    "After running the above command line, you will observe a new folder NeMoTTS_sup_data/pitch and printouts of pitch statistics like below. Specify these values to the FastPitch training configurations. We will be there in the following section.\n",
+    "```bash\n",
+    "PITCH_MEAN=175.48513793945312, PITCH_STD=42.3786735534668\n",
+    "PITCH_MIN=65.4063949584961, PITCH_MAX=270.8517761230469\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c9a73569",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!cd {codedir} && python scripts/dataset_processing/tts/extract_sup_data.py \\\n",
+    "    manifest_filepath={valid_manifest} \\\n",
+    "    sup_data_path={suppdir} \\\n",
+    "    dataset.sample_rate={sample_rate} \\\n",
+    "    dataset.n_fft=2048 \\\n",
+    "    dataset.win_length=2048 \\\n",
+    "    dataset.hop_length=512"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "caa2da01",
+   "metadata": {},
+   "source": [
+    "## c. Training"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3d6d51a1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "phoneme_dict_path = os.path.abspath(os.path.join(codedir, \"scripts\", \"tts_dataset_files\", \"cmudict-0.7b_nv22.10\"))\n",
+    "heteronyms_path = os.path.abspath(os.path.join(codedir, \"scripts\", \"tts_dataset_files\", \"heteronyms-052722\"))\n",
+    "\n",
+    "# Copy and Paste the PITCH_MEAN and PITCH_STD from previous steps (train_manifest) to overide pitch_mean and pitch_std configs below.\n",
+    "PITCH_MEAN=175.48513793945312\n",
+    "PITCH_STD=42.3786735534668"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "eecad436",
+   "metadata": {},
+   "source": [
+    "### Important notes\n",
+    "* `+init_from_ptl_ckpt`: initialize with a multi-speaker FastPitch checkpoint\n",
+    "* `~model.speaker_encoder.lookup_module`: remove the pre-trained looked-up speaker embedding\n",
+    "* Other optional arguments based on your preference:\n",
+    "    * batch_size\n",
+    "    * exp_manager\n",
+    "    * trainer"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "78680a01",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Normally 100 epochs (15 mins)\n",
```

Comment:
I would avoid putting the minute estimations since it is highly dependent on GPU type.